specs/eventing/control-plane.md

```diff
@@ -0,0 +1,985 @@
+    <td>RECOMMENDED</td>
+  </tr>
+  <tr>
+    <td><code>backoffDelay</code></td>
+    <td>string</td>
+    <td>The initial delay when retrying delivery.</td>
+    <td>RECOMMENDED</td>
+  </tr>
+  <tr>
+    <td><code>backoffPolicy</code></td>
+    <td>enum<br/>("linear", "exponential")</td>
+    <td>Retry timing scaling policy. Linear policy uses the same <code>backoffDelay</code> for each attempt; Exponential policy uses 2^N multiples of <code>backoffDelay</code></td>
+    <td>RECOMMENDED</td>
+  </tr>
+</table>
+
+### KReference
+
+KReference is a lightweight version of kubernetes
+[`v1/ObjectReference`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectreference-v1-core)
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>apiVersion</code></td>
+    <td>string</td>
+    <td>ApiVersion of the target reference.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>kind</code></td>
+    <td>string</td>
+    <td>Kind of the target reference.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>namespace</code></td>
+    <td>string</td>
+    <td>Namespace of the target resource.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>address</code></td>
+    <td>string</td>
+    <td>Address used to deliver events to the Broker.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### PhysicalSubscriptionStatus
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>subscriberUri</code></td>
+    <td>URL (string)</td>
+    <td>Resolved address of the <code>spec.subscriber</code>.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>replyUri</code></td>
+    <td>URL (string)</td>
+    <td>Resolved address of the <code>spec.reply</code>.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>deadLetterSinkUri</code></td>
+    <td>URL (string)</td>
+    <td>Resolved address of the <code>spec.delivery.deadLetterSink</code>.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### TriggerFilter
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>attributes</code></td>
+    <td>map[string]string</td>
+    <td>Event filter using exact match on event context attributes. Each key in the map is compared with the equivalent key in the event context. An event passes the filter if the event attribute values are equal to the specified values, or if the value in the map is the empty string (which is treated as matching all values but requiring the key to be present).</td>
```

Comment:
I think we need some RFC keywords in here.
MUST only match if the specified attribute name/value are an exact match.... that kind of thing.  Case?

Reply:
And I assume all name/value pairs MUST match, it's not an "OR"

Reply:
Yes, it's an AND. Clarified both, I hope.